system/draft/TestProductionSuite.py

```diff
@@ -33,7 +33,8 @@ class TestProductionSuite:
         )
 
         # put both genesis files into extra nodes
-        assert all([c.exec_run(['mkdir', GENESIS_PATH], user='indy').exit_code == 0 for c in extra_containers])
+        print([c.exec_run(['mkdir', GENESIS_PATH], user='indy') for c in extra_containers])
```

Comment:
Why assert has been replaced by print?

Reply:
We have post install scripts in sovrin that create this folder but does not have them in indy-node so this assert will fail against sovrin setup.